agent.go

```diff
@@ -52,7 +52,15 @@ func (eh epHealth) String() string {
 }
 
 func clusterHealth(gcfg globalConfig) ([]epHealth, error) {
-	eps, err := endpointsFromCluster(gcfg)
+	var eps []string
+	var err error
+
+	if gcfg.skipHealthcheckClusterEndpoints {
+		eps, err = endpointsFromCmd(gcfg)
```

Comment:
```suggestion
		eps, err = endpoints(gcfg)
```

also we need to call `endpointsFromCmd` in the following line,

https://github.com/ahrtr/etcd-defrag/blob/fc3e35bc3b8e8643a6a17096e73ab9da02273729/endpoints.go#L40